src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -2267,6 +2267,72 @@ paths:
 ##############################################################################
 ## SEARCH API STANDARD ENDPOINTS
 ##############################################################################
+  /api/repository/v1/search:
+    get:
+      tags:
+        - snapshots
+        - repository
+      description: >
+        Returns a list of all snapshots a user has access to and their search criteria
+      operationId: enumerateSnapshotSearch
+      parameters:
+        - name: offset
```

Comment:
I think for MVP we won't need any of these parameters, and I'd like to avoid adding any features until they're needed. So unless we need these immediately, I think it would be better to make this endpoint take no parameters.